__docs__/wonder-blocks-tabs/navigation-tabs-variants.stories.tsx

```diff
@@ -141,6 +227,58 @@ export const Active: Story = {
     parameters: {pseudo: {hover: true, active: true}},
 };
 
+export const Zoom: Story = {
+    render: (args) => (
+        <div>
+            <AllVariants rows={rows} columns={columns} layout="list">
+                {(props, name) => (
+                    <View style={styles.container}>
+                        <NavigationTabs
+                            {...args}
+                            {...props}
+                            aria-label={name}
+                        />
+                    </View>
+                )}
+            </AllVariants>
+            <div dir="rtl">
+                <AllVariants rows={rtlRows} columns={columns} layout="list">
+                    {(props, name) => (
+                        <View style={styles.container}>
+                            <NavigationTabs
+                                {...args}
+                                {...props}
+                                aria-label={`${name} RTL`}
+                            />
+                        </View>
+                    )}
+                </AllVariants>
+            </div>
+        </div>
+    ),
+    parameters: {
+        a11y: {
+            config: {
+                rules: [
+                    // Disabling warning: "Element's background color could not
```

Comment:
I'm curious about this one, as I wrote part of the overlapping logic in the color-contrast rule! It looks like the Axe extension doesn't surface incompletes anymore, and I don't see any Needs Review items in Accessibility Insights either. Maybe Storybook is using an older axe-core API version?

Reply:
It was showing as an incomplete in the Storybook add on! Any tips on how to address this? 

<img width="1725" alt="image" src="https://github.com/user-attachments/assets/40ca2fd1-a78d-48c5-a15b-3fc401da6963" />

I also see it in the labels for the story! Might be related to setting the CSS zoom property?
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/b76c641f-b434-43a7-bb33-b7bbc21b67d8" />